sources/packages/backend/libs/utilities/src/_tests_/unit/string-utils.translateToASCII.spec.ts

```diff
@@ -36,4 +36,19 @@ describe("StringUtils-convertToASCII", () => {
     // Assert
     expect(translatedData).toBeNull();
   });
+
+  it("Should replace ASCII control characters (0-31) with '?'", () => {
```

Comment:
The test description should try to keep a "Should"/"When" pattern and have a period at the end.